deploy/modules/aliyun/tidb-operator/operator.tf

```diff
@@ -25,13 +25,18 @@ resource "null_resource" "setup-env" {
     # it manually and the resource namespace & name are hard-coded by convention
     command     = <<EOS
 kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/${var.operator_version}/manifests/crd.yaml
+kubectl apply -f ${path.module}/manifest/global-job-controller.yaml
 kubectl apply -f ${path.module}/manifest/alicloud-disk-storageclass.yaml
 echo '${data.template_file.local-volume-provisioner.rendered}' | kubectl apply -f -
 kubectl patch -n kube-system daemonset flexvolume --type='json' -p='[{"op":"replace", "path": "/spec/template/spec/tolerations", "value":[{"operator": "Exists"}]}]'
 helm init
 until helm ls; do
   echo "Wait tiller ready"
 done
+until kubectl get crd globaljobs.jobs.aliyun.com; do
+  echo "Wait for globaljobs crd ready"
+done
```

Comment:
use `kubectl wait --for=condition=Established ...`, example: https://github.com/pingcap/tidb-operator/pull/997/files#diff-67dacabf080ca4dc8e95d2acf12ee36bR388

- kubectl wait is a simple way to wait for conditions
- crd can only be used when it's established